packages/tools/devtools/devtools-view/src/components/Menu.tsx

```diff
@@ -411,21 +444,34 @@ interface ContainersMenuSectionProps {
 	 * @remarks Passing `undefined` clears the selection.
 	 */
 	selectContainer(containerKey: ContainerKey | undefined): void;
+
+	/**
+	 * Whether to show as "Data Objects" instead of "Containers".
+	 */
+	isDataObjects?: boolean;
```

Comment:
Instead of passing this in, it might be cleaner to just pass in the section label.